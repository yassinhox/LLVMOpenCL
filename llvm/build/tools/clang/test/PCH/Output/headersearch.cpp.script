rm -rf /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_moved &&
mkdir -p /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/sub /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/sub2 &&
echo 'struct orig_sub{char c; int i; };' > /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/sub/orig_sub.h &&
echo 'void orig_sub2_1();' > /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/sub2/orig_sub2_1.h &&
echo '#include "orig_sub2_1.h"' > /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/sub2/orig_sub2.h &&
echo 'template <typename T> void tf() { orig_sub2_1(); T::foo(); }' >> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/sub2/orig_sub2.h &&
echo 'void foo() {}' > /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/tmp2.h &&
echo '#include "tmp2.h"' > /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/all.h &&
echo '#include "sub/orig_sub.h"' >> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/all.h &&
echo '#include "orig_sub2.h"' >> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/all.h &&
echo 'int all();' >> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/all.h &&
cd /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig && /home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -x c++ -emit-pch -o all.h.pch -Isub2 all.h &&
cp -pR /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_moved &&
/home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -include-pch all.h.pch -I/home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_moved -I/home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_moved/sub2 -Wpadded -emit-obj -o /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.o /home/jazouani/llvm/llvm-3.2/tools/clang/test/PCH/headersearch.cpp 2> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.stderr &&
grep 'struct orig_sub' /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.stderr &&
not /home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -DREDECL -include-pch all.h.pch -I/home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_moved -I/home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_moved/sub2 -emit-obj -o /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.o /home/jazouani/llvm/llvm-3.2/tools/clang/test/PCH/headersearch.cpp 2> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.stderr &&
grep 'void foo' /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.stderr &&
not /home/jazouani/llvm/build/Release+Asserts/bin/clang -cc1 -internal-isystem /home/jazouani/llvm/build/Release+Asserts/bin/../lib/clang/3.2/include -DINSTANTIATION -include-pch all.h.pch -I/home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_moved -I/home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_moved/sub2 -emit-obj -o /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.o /home/jazouani/llvm/llvm-3.2/tools/clang/test/PCH/headersearch.cpp 2> /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.stderr &&
grep 'orig_sub2_1' /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp.stderr
