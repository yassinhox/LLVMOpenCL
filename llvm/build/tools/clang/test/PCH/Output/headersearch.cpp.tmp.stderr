In file included from /home/jazouani/llvm/llvm-3.2/tools/clang/test/PCH/headersearch.cpp:1:
In file included from /home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/all.h:3:
/home/jazouani/llvm/build/tools/clang/test/PCH/Output/headersearch.cpp.tmp_orig/sub2/orig_sub2.h:2:50: error: type 'int' cannot be used prior to '::' because it has no members
template <typename T> void tf() { orig_sub2_1(); T::foo(); }
                                                 ^
/home/jazouani/llvm/llvm-3.2/tools/clang/test/PCH/headersearch.cpp:42:3: note: in instantiation of function template specialization 'tf<int>' requested here
  tf<int>();
  ^
1 error generated.
